packages/web-react/src/components/FileUploader/stories/FileUploader.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { ComponentStory } from '@storybook/react';
+import { SpiritFileUploaderAttachmentProps, FileUploaderQueueItemProps } from '../../../types';
+import FileUploader from '../FileUploader';
+import FileUploaderInput from '../FileUploaderInput';
+import FileUploaderList from '../FileUploaderList';
+import FileUploaderAttachment from '../FileUploaderAttachment';
+
+const Story: ComponentStory<typeof FileUploader> = () => {
+  const [queue, setQueue] = useState<FileUploaderQueueItemProps[]>([]);
+
+  const attachmentTemplate = (props: SpiritFileUploaderAttachmentProps) => (
+    <FileUploaderAttachment key={props.id} {...props} />
+  );
+
+  const onDismissHandler = (name: string) => {
+    const prevState = [...queue];
+    const index = prevState.findIndex((item) => item.key === name);
+    prevState.splice(index, 1);
+    setQueue(prevState);
+  };
+
+  const addToQueueHandler = (key: string, file: File) => {
+    setQueue((prevState) => [...prevState, { key, file }]);
+  };
+
+  return (
+    <FileUploader
+      id="fileUploaderExample"
+      onDismiss={onDismissHandler}
+      fileQueue={queue}
+      addToQueue={addToQueueHandler}
+    >
+      <FileUploaderInput
+        id="fileUploaderExampleInput"
+        name="attachments"
+        label="Label"
+        linkText="Upload your file(s)"
+        labelText="or drag and drop here"
+        helperText="Max file size is 10 MB"
+        validationMessage="Validation message"
+        fileQueueLimit={2}
+        fileQueue={queue} // TODO: to sem normálně nepatří
+        onError={(error) => console.log('My error log', error)}
+      />
+      <FileUploaderList
+        id="fileUploaderExampleList"
+        label="Attachments"
+        inputName="attachments"
+        attachmentTemplate={attachmentTemplate}
```

Comment:
I am thinking about better naming here. Actually, the input is a component and it is also the render prop. The `template` does not suit here well.

Reply:
Maybe `attachmentComponent` ?

Reply:
done